qutebrowser/completion/models/base.py

```diff
@@ -44,6 +44,17 @@ class BaseCompletionModel(QStandardItemModel):
         COLUMN_WIDTHS: The width percentages of the columns used in the
                         completion view.
         DUMB_SORT: the dumb sorting used by the model
+
+    Attributes:
+        filtered_out_cache:
```

Comment:
I *think* it should be fine as the cache is reset with an empty pattern anyways.